src/HieDb/Create.hs

```diff
@@ -224,17 +224,16 @@ addRefsFromLoaded
   :: MonadIO m
   => HieDb -- ^ HieDb into which we're adding the file
   -> FilePath -- ^ Path to @.hie@ file
-  -> SourceFile -- ^ Path to .hs file from which @.hie@ file was created
-                -- Also tells us if this is a real source file?
-                -- i.e. does it come from user's project (as opposed to from project's dependency)?
+  -> SourceFile -- ^ Ignored but maintained for backward compatibility - pass in a (FakeFile Nothing)
```

Comment:
Maintained for backwards compatibility, this should probably be removed on the next major version bump